srcpkgs/kindd/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'kindd'
+pkgname=kindd
+version=2.1.0
+revision=1
+wrksrc="kindd-${version}"
+build_style=cmake
+hostmakedepends="cmake pkg-config"
```

Comment:
```suggestion
hostmakedepends="cmake pkg-config qt5-qmake"
```
cmake just likes to use qmake to detect some stuff, idk what exactly it does with it out of my head.

Reply:
Yeah, i've also added `qt5-host-tools` due to some complains that arm ARCH about:
>  The imported target "Qt5::Core" references the file

     "/usr/lib/qt5/bin/moc"